js/mapbox-gl.js

```diff
@@ -13,6 +13,8 @@ mapboxgl.Attribution = require('./ui/control/attribution');
 mapboxgl.Popup = require('./ui/popup');
 mapboxgl.Marker = require('./ui/marker');
 
+mapboxgl.WorkerPool = require('./util/worker_pool');
```

Comment:
I'm 👎  on adding this to our public interface, especially if only to expose a constant.